content/en/docs/Concepts/olm-architecture/operator-catalog/creating-an-update-graph.md

```diff
@@ -118,24 +106,149 @@ version.
 ### Cross channel updates
 
 Cross channel updates are not possible today in an automated way. In order for your subscription
-to switch channels, the cluster admin must manually change the `channel` field in the subcription
+to switch channels, the cluster admin must manually change the `channel` field in the subscription
 object.
 
 Changing this field does not always result in receiving updates from that channel. For that to
 occur, updates paths must be available from the given version to versions in the new channel.
 
 #### If using replaces
 
-The CSV name currently installed must be in the `replaces` field of a CSV in the new channel.
+The CSV name currently installed must be in the `replaces` field of an entry in the new channel.
```

Comment:
It may actually be important in this case to keep using CSV name, because that's actually what OLM is using to decide what the upgrade candidates are when a CSV is installed on the cluster.